exporters/readers/s3_reader.py

```diff
@@ -242,7 +242,6 @@ def get_read_streams(self):
             file_obj = urlopen(key.generate_url(S3_URL_EXPIRES_IN))
             yield Stream(file_obj, key_name, key.size)
 
-    @retry_long
     def read_lines_from_keys(self):
         for current_key in self.keys:
             self.current_key = current_key
```

Comment:
Problem with this approach is that we "might" have read some batches from a file before it fails, and the retry would read them again. Maybe we should use long retries at key.read() method, as we expect that if a read fails, next read should give us the proper chunk.